fastai/dataset.py

```diff
@@ -90,6 +102,10 @@ def create_val(path, r):
     copy_or_move_with_subdirs(subdirs, path, val_path, r, move=True)
 
 def copy_or_move_with_subdirs(subdir_lst, src, dst, r, move=False):
+    '''
```

Comment:
Is this used anywhere? I don't know what it is. I don't think your current docs are correct - probably best not to add a doc if you don't know what something is. Maybe instead just add `TODO: doc` as the docstring if you don't know?

(For this one though, if it's not used anywhere in fastai we can remove it - but be sure to check both fastai and notebook dirs)